docs/DeveloperGuide.md

```diff
@@ -337,6 +337,41 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
+**Use case: Edit an employee**
+
+**MSS**
+
+1. User request to list employees
+2. HeRon shows list of employees
+3. User request to edit a specific employee on the list
```

Comment:
Just a small thing, but maybe it would be good to standardise the tense with the other examples, specifically "User requests" rather than "request"?